### PR TITLE
Introduction post about the blog

### PR DESCRIPTION
**Pull request by Fgerthoffert**
First draft, comments are welcome. @btiernay @shanewilson @andricDu

### CODE REVIEW

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Welcome to our blog, welcome in our world. We are a team of software engineers, infrastructure specialists and bioinformaticians building infrastructures and tools used by cancer reasearchers around the world. 
+
+This blog is our opportunity to share with the world how we tackle challenges, find solutions, share our knowledge. All of this applicable not only to cancer research, but to IT in general.
+
+## An idea
+
+Discussions on this topic were triggered by a couple of very interesting sprint demos in which team members showed us solutions and walked us through challenge they faced. 
+
+With very limited documentation available online on those particular topics, it was clear we had knowlegde worth sharing with the world.
```
Comment (andricDu) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: `knowlegde` -> `knowledge`

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+This blog is our opportunity to share with the world how we tackle challenges, find solutions, share our knowledge. All of this applicable not only to cancer research, but to IT in general.
+
+## An idea
+
+Discussions on this topic were triggered by a couple of very interesting sprint demos in which team members showed us solutions and walked us through challenge they faced. 
+
+With very limited documentation available online on those particular topics, it was clear we had knowlegde worth sharing with the world. 
+
+We have a mandate to share and open-source our software, but futher than that, we believe that sharing how we got to that new software version, this cloud infrastructure, or this particular piece of implementation, is also very relevant.
```
Comment (andricDu) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: `futher` -> `further`

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## A technical platform
+ 
+Implementing a blog comes with its own sets of challenges, from ease of contribution to maintability.
```
Comment (andricDu) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: `maintability` -> `maintainability`

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Our group has expertise with 2 types of content-publishing platforms; CMS (such as Drupal) and Statics website generators (such as MKDocs or Jekyll).
+CMS platforms provide greater flexibility and capabilities, but are also much more complex to maintain (especially from a security standpoint). Static website generators are easier to operate (no databases, no server-side code), with content being added following a git workflow developers are familiar with.
+
+We decided to move ahead with Jekyll as a static website generator because of its native support by GiHub pages. We can then rely on Github for building, hosting and maintaining the platform. The team will only need to focus on the content itself.
```
Comment (andricDu) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: `GiHub` -> `GitHub`

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+date: 2016-10-30
+categories: francois_gerthoffert
+tags:
+    - Blog
+    - Talk
+teaser:
+    info: Welcome to our blog, welcome in our world. We are a team of software engineers, infrastructure specialists and bioinformaticians building infrastructures and tools used by cancer reasearchers around the world. 
+---
+
+Welcome to our blog, welcome in our world. We are a team of software engineers, infrastructure specialists and bioinformaticians building infrastructures and tools used by cancer reasearchers around the world.
```
Comment (andricDu) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: `reasearchers` -> `researchers`

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+title:  "Hello World"
+breadcrumb: true
+author: francois_gerthoffert
+date: 2016-10-30
+categories: francois_gerthoffert
+tags:
+    - Blog
+    - Talk
+teaser:
+    info: Welcome to our blog, welcome in our world. We are a team of software engineers, infrastructure specialists and bioinformaticians building infrastructures and tools used by cancer reasearchers around the world.
```
Comment (andricDu) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: `reasearchers` -> `researchers`

**Comment by andricDu** [comment]
I like it! Short and sweet!

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md (hunk 6 above)
Comment (cy) [reply _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: `[...], welcome to our world.`

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+We decided to move ahead with Jekyll as a static website generator because of its native support by GiHub pages. We can then rely on Github for building, hosting and maintaining the platform. The team will only need to focus on the content itself.
+
+## A blog
+
+It took us a couple of months to get there, the blog having (obviously) a much lower priority compared to our other activities. But after running ideas through our design team, we were ready to start implementing the technical platform.
+
+Today the blog is open to the world, the team will be contributing regularly and welcome external contributions. 
+
+This blog is yours !
```
Comment (cy) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: `yours !` there's an extra space after yours

**Review by cy — COMMENTED** [review]
🖋⭐️

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+## An idea
+
+Discussions on this topic were triggered by a couple of very interesting sprint demos in which team members showed us solutions and walked us through challenge they faced. 
+
+With very limited documentation available online on those particular topics, it was clear we had knowlegde worth sharing with the world. 
+
+We have a mandate to share and open-source our software, but futher than that, we believe that sharing how we got to that new software version, this cloud infrastructure, or this particular piece of implementation, is also very relevant.
+
+## A technical platform
```
Comment (shanewilson) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md:27]: I would probably drop this section. Drupal and Jekyll are super boring. Also saying we have expertise in these makes it sounds like we use them daily.

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+Implementing a blog comes with its own sets of challenges, from ease of contribution to maintability.
+
+Our group has expertise with 2 types of content-publishing platforms; CMS (such as Drupal) and Statics website generators (such as MKDocs or Jekyll).
+CMS platforms provide greater flexibility and capabilities, but are also much more complex to maintain (especially from a security standpoint). Static website generators are easier to operate (no databases, no server-side code), with content being added following a git workflow developers are familiar with.
+
+We decided to move ahead with Jekyll as a static website generator because of its native support by GiHub pages. We can then rely on Github for building, hosting and maintaining the platform. The team will only need to focus on the content itself.
+
+## A blog
+
+It took us a couple of months to get there, the blog having (obviously) a much lower priority compared to our other activities. But after running ideas through our design team, we were ready to start implementing the technical platform.
```
Comment (shanewilson) [inline _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: This sounds kind of dismissive.  Could probably remove this and just have the sentence below which is more positive

Comment (Fgerthoffert) [reply _posts/francois_gerthoffert/2016-10-30-Hellow-World.md]: Right, it was not really adding value

File: _posts/francois_gerthoffert/2016-10-30-Hellow-World.md (hunk 8 above)
Comment (Fgerthoffert) [reply _posts/francois_gerthoffert/2016-10-30-Hellow-World.md:27]: Actually, we do use those daily. Not so much on the icgc, but Collaboratory and GDC CMSs are actively being updated those days.

**Pull request merged**